CIPs/cip-137.md

```diff
@@ -0,0 +1,200 @@
+---
+cip: 137
+title: Ceramic API
+author: Danny Browning <dbrowning@3box.io>, Nathaniel Cook <nathaniel@3box.io>, Aaron Goldman <aaron@3box.io>, Joel Thorstensson <joel@3box.io>
+discussions-to: <URL to a topic under the CIP category on the Ceramic forum: https://forum.ceramic.network/c/cips>
+status: Draft
+category: Interface
+created: 2023-05-28
+edited: 2023-05-28
+requires: 124
+---
+
+
+## Simple Summary
+<!--Provide a simplified and layman-accessible explanation of the CIP.-->
+An API for reading and writing events on Ceramic event streams.
+
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+TODO
+
+
+## Motivation
+<!--Motivation is critical for CIPs that want to change the Ceramic protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the CIP solves. CIP submissions without sufficient motivation may be rejected outright.-->
+In order to enable databases other than ComposeDB to be built on top of Ceramic, an API to read and write directly on event streams is needed. This also enables developers to get direct access to the event streams underlaying ComposeDB.
+
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature.-->
+
+This is a work in progress specification that introduces a JSON-RPC interface for interacting directly with event streams on a node.
+
+
+
+### `ceramic_subscribe`
```

Comment:
So what you are subscribing to is a separator-key/value, in the example it happens to be a model. It can definitely be possible to subscribe to specific streams within a separator!

Reply:
Let's say stream-bar is from model-bar. So the "topic" could be "model-foo" or "stream-bar" or explicitly "model-foo/stream-bar"?